src/app/modules/group/components/member-list/member-list.component.ts

```diff
@@ -117,6 +119,7 @@ export class MemberListComponent implements OnChanges, OnDestroy {
 
   ngOnDestroy(): void {
     this.dataFetching.complete();
+    this.refresh$.complete();
```

Comment:
since you complete other subjects too, can you complete `removalInProgress$` as well :pray: ?

Reply:
Fixed